lib/fastly.rb

```diff
@@ -690,12 +690,12 @@ def self.load_config(file)
     options = {}
     return options unless File.exist?(file)
 
-    File.open(file, 'r') do |infile|
+    File.open(file, "r") do |infile|
       while line = infile.gets
         line.chomp!
-        next if line =~ /^#/
-        next if line =~ /^\s*$/
-        next unless line =~ /=/
+        next if /^#/.match?(line)
```

Comment:
These changes seems to have created some `UndefinedMethod` failures, will explore.